src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -122,17 +122,17 @@ def profile_setup():
     """Set up a new profile."""
 
 
-@verdi_profile.command('configure-rabbitmq')  # type: ignore[arg-type]
+@verdi_profile.command('configure-rabbitmq')
+@click.pass_context
```

Comment:
Not quite sure why, but moving `pass_context` here fixed the following error:

```
src/aiida/cmdline/commands/cmd_profile.py:135: error: Argument 1 to "pass_context" has incompatible type "Callable[[Any, Any, Any, Any, KwArg(Any)], Any]"; expected "Callable[[Context, Any, Any, Any, KwArg(Any)], Any]"  [arg-type]
src/aiida/cmdline/commands/cmd_profile.py:135: note: This is likely because "profile_configure_rabbitmq" has named arguments: "ctx". Consider marking them positional-only
```

Reply:
You forgot this one. Maybe not change the order, change the order of decorator change the behavior, correct?

Reply:
Ah, yes, thanks for the remainder. As you noticed, in the end I just read the error message more carefuly and marked ctx as a position-only argument. I am not sure why it is necessary here, and not in the other places? Perhaps because we have the `**kwargs` here. But in any case, this feels like a safe change.